index.md

```diff
@@ -4,23 +4,38 @@ layout: default
 
 _“Best practice templates and writing instructions for documenting open source software.”_
 
+### What?
 
-### Download
+Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. 
 
-* The [0.1 alpha straw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.
-* The latest development version of templates are available from the [github main branch](https://github.com/thegooddocsproject/templates).
+We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our logic, referencing research where available.
 
 ### Why?
```

Comment:
For a vision, I thinks this should be presented in the Why/How/What/(Who) format, in line with Simon Sinek's reasoning:
https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action?language=en

Reply:
Nice Ted talk. I'll see what I can do.